core/src/main/java/com/graphhopper/routing/ch/EdgeBasedPathCH.java

```diff
@@ -30,19 +26,7 @@ protected ShortcutUnpacker getShortcutUnpacker(Graph routingGraph, final Weighti
             @Override
             public void visit(EdgeIteratorState edge, boolean reverse, int prevOrNextEdgeId) {
                 distance += edge.getDistance();
-                // a one-way loop that is not a shortcut cannot possibly be read in the 'right' direction, because
```

Comment:
I have changed `calcMillis` such that loops are read in fwd direction only so this is no longer needed here. This change is not strictly related to removing the u-turn mode though.